addons/containerd/template/base/Manifest

```diff
@@ -0,0 +1,8 @@
+image pause k8s.gcr.io/pause:3.1
+image pause k8s.gcr.io/pause:3.2
```

Comment:
why two pause versions?

Reply:
This should be 3.1 for containerd 1.3 and 3.2 for 1.4. I'll update.

Containerd overrides the pod sandbox image so depending on the version of Kubernetes installed, the Kubernetes airgap package may not contain the pause version that containerd actually uses.